strings/naive_string_search.py

```diff
@@ -39,4 +39,4 @@ def naive_pattern_search(s: str, pattern: str) -> list:
 
 if __name__ == "__main__":
     assert naive_pattern_search("ABCDEFG", "DE") == [3]
-    print(f"{naive_pattern_search('ABAAABCDBBABCDDEBCABC', 'ABC') = }")
+    print(f"{naive_pattern_search('ABAAABCDBBABCDDEBCABC', 'ABC')}")
```

Comment:
Why break this?

Reply:
I was getting error in pycharm 'unexpected expression part' before running a program.

Reply:
Pycharm is not upgraded to Python 3.8 and 3.9 which support this functionality.

Reply:
I have fixed this